django-stubs/core/mail/message.pyi

```diff
@@ -60,15 +65,15 @@ class EmailMessage:
     def __init__(
         self,
         subject: str = ...,
-        body: Optional[str] = ...,
+        body: str = ...,
         from_email: Optional[str] = ...,
-        to: Optional[Union[Sequence[str], str]] = ...,
-        bcc: Optional[Union[Sequence[str], str]] = ...,
+        to: Optional[Sequence[str]] = ...,
+        bcc: Optional[Sequence[str]] = ...,
         connection: Optional[Any] = ...,
-        attachments: Optional[Union[List[Tuple[str, Union[str, bytes], str]], List[MIMEText]]] = ...,
+        attachments: Optional[Sequence[Union[MIMEBase, _AttachmentTuple]]] = ...,
         headers: Optional[Dict[str, str]] = ...,
-        cc: Optional[Union[Sequence[str], str]] = ...,
-        reply_to: Optional[Union[List[Optional[str]], str]] = ...,
+        cc: Optional[Sequence[str]] = ...,
+        reply_to: Optional[Sequence[str]] = ...,
```

Comment:
`reply_to` might also be a `Sequence[None]`, but from the implementation it looks like an incorrect test
https://github.com/django/django/blob/master/tests/mail/tests.py#L101

Reply:
I think these two are tests problem

>         'List item 1 has incompatible type "None"; expected "str"',
>        'Incompatible return value type (got "None", expected "bool")',

will try to lend a patch into the django. We also can loose requirements here but i'm not sure, should we loose types if tests are using trash data as input for testing purposes only?

Reply:
> 'List item 1 has incompatible type "None"; expected "str"',

It's for `body` and `reply_to` I assume? Ignoring it like you did seems fine and actually the right call.  

What about the second line? Which code does it correspond to? 
> 'Incompatible return value type (got "None", expected "bool")',

Reply:
https://github.com/django/django/blob/master/tests/mail/tests.py#L1611 this line, the fix should be lambda: False

Reply:
Seems reasonable, thanks.